AlfrescoApp/Utils/Utility.m

```diff
@@ -881,11 +881,24 @@ + (NSDictionary *)metadataByAddingGPSToMetadata:(NSDictionary *)metadata
     NSDictionary *gpsDictionary = @{(NSString *)kCGImagePropertyGPSLatitude : [NSNumber numberWithFloat:fabs(coordinates.latitude)],
                                     (NSString *)kCGImagePropertyGPSLatitudeRef : ((coordinates.latitude >= 0) ? @"N" : @"S"),
                                     (NSString *)kCGImagePropertyGPSLongitude : [NSNumber numberWithFloat:fabs(coordinates.longitude)],
-                                    (NSString *)kCGImagePropertyGPSLongitudeRef : ((coordinates.longitude >= 0) ? @"E" : @"W")};
+                                    (NSString *)kCGImagePropertyGPSLongitudeRef : ((coordinates.longitude >= 0) ? @"E" : @"W")
+    };
     
     [returnedMetadata setValue:gpsDictionary forKey:(NSString *)kCGImagePropertyGPSDictionary];
     
     return returnedMetadata;
 }
 
+
++ (NSDictionary *)metadataByAddingOrientation:(NSInteger)orientation toMetadata:(NSDictionary *)metadata
+{
+    NSMutableDictionary *returnedMetadata = [metadata mutableCopy];
+    
+//    NSDictionary *orientationDictionary = @{(NSString *)kCGImagePropertyOrientation : [NSNumber numberWithInt:orientation]};
```

Comment:
If not required, you can remove the commented code or add a document note to it for future reference